### PR TITLE
FIX: improve logging

### PR DESCRIPTION
**Pull request by kammala**

### CODE REVIEW

File: time_execution/backends/elasticsearch.py [hunk 1]
```diff
@@ -119,7 +119,4 @@ def bulk_write(self, metrics):
         for metric in metrics:
             actions.append({'index': {'_index': index, '_type': self.doc_type}})
             actions.append(metric)
-        try:
-            self.client.bulk(actions)
-        except TransportError as exc:
-            logger.warning('bulk_write metrics %r failure %r', metrics, exc)
+        self.client.bulk(actions)
```
Comment (igbun) [inline time_execution/backends/elasticsearch.py]: Can you elaborate why this change in `bulk_write` and not in `write`?

Comment (kammala) [reply time_execution/backends/elasticsearch.py]: well, right now we have a problem with the error being logged twice: once from `ElasticsearchBackend.bulk_write` and next time from [`ThreadedBackend.worker`](https://github.com/kpn-digital/py-timeexecution/blob/master/time_execution/backends/threaded.py#L95).

since `ThreadedBackend` catches all kinds of error, I would prefer to have logging there.

but thanks for pointing out, I will also remove the same thing from `KafkaBackend`

Comment (kammala) [reply time_execution/backends/elasticsearch.py]: nope, that's not truth, because we do not reraise the error.

removed that part.
